helper.py

```diff
@@ -140,95 +140,11 @@
 for zero in number_zero:
   print(zero)
 
-my_dict ={ "1" : [
-  "  ____   ",
-  " /    |  ",
-  "/_/|  |  ",
-  "   |  |  ",
-  "   |  |  ",
-  "  _|  |_ ",
-  " |______|",
-], "2" : [
-"   _____    ",
-" /  __  |  ",
-"/__/ / /   ",
-"    / /    ",
-"   / /     ",
-"  / /____   ",
-" |_______|  ",
-], "3" : [
-"  ________    ",
-" |_____  |   ",
-"    ___| |   ",
-"   |___  |   ",          
-"       | |   ",    
-"  _____| |   ",
-" |_______|   ", 
-], "4" : [
-" __    __  ",
-"|  |  |  | ",
-"|  |  |  | ",
-"|  |__|  | ",
-"|_____|  | ",
-"      |  | ",
-"      |__| ",
-], "5" :[
-  " ______   ",
-  "|  ____|  ",
-  "|  |__    ",
-  "|____ \\  ",
-  " _   \\ | ",
-  "| |__| |  ",
-  " \____/   ",
-],"6" : [
-"  _______  ",
-  "|  ____| ",
-  "|  |__    ",
-  "| __  \\  ",
-  "| | \\  | ",
-  "| |__| |  ",
-  " \____/   ",
-], "7" :[
-"  _______    ",
-" /____  /    ",
-"     / /     ",
-"    / /      ",
-"   / /       ",
-"  / /        ",
-" /_/         ",
-], "8" :[
-  "  _____  ",
-  " /  _  \\ ",
-  "/  |_|  \\",
-  "\\       /",
-  "/   _   \\",
-  "\\  |_|  /",
-  " \\_____/ ",
-], "9" :[
-  "  _______ ",
-  " /   _   |",
-  "|   |_|  |",
-  " \\___    |",
-  "     |   |",
-  "     |   |",
-  "     |___|",
-],"0" :[
-
-"   ____     ",
-"  / __ \\   ",
-" | /  \\ |  ",
-" | |  | |  ",
-" | |  | |  ",
-" | \\__/ |  ",
-"  \\____/   ",
-]      
+my_dict ={ 
+  "1" :number_one,, "2" : number_two, "3" : number_three, "4" : number_four, "5" : number_five,"6" : number_six, "7" : number_seven, "8" :number_eight,
+  , "9" :number_nine,"0" :number_zero 
 }
```

Comment:
@Anahis259 , you are almost there, Two more changes:

1) Break this dictionary into multiple lines
    (fewer lines of code no always mean better code, your main focus is that your code is readable.)
```python
my_dict ={ 
"1" :number_one,
"2" : number_two, 
"3" : number_three, 
....
# The rest of the lines of code.
....
"8" :number_eight,
"9" :number_nine,
"0" :number_zero,
}
```

2) Add a meaningful name for the dictionary. 
    (`my_dict` is too vague)

```python
form_num_str_to_num_dict_mapper ={ 
"1" :number_one,
"2" : number_two, 
"3" : number_three, 
....
# The rest of the lines of code.
....
"8" :number_eight,
"9" :number_nine,
"0" :number_zero,
}
```